index.js

```diff
@@ -182,6 +182,11 @@ exports.evm2wast = function (evmCode, opts = {
     const opint = evmCode[pc]
     const op = opcodes(opint)
 
+    // creates a stack trace
+    if (opts.stackTrace) {
+      segment += `(call $stackTrace (i32.const ${pc}) (i32.const ${opint}) (i32.const ${gasCount}) (get_global $sp))\n`
```

Comment:
The linked issue shows that `gasCount` here should be the actual gas usage of the instruction, but here this value is always 0 after this change.